pyrs/core/workspaces.py

```diff
@@ -895,14 +895,30 @@ def save_wavelength(self, hidra_project):
         if self._wave_length is not None:
             hidra_project.write_wavelength(self._wave_length)
 
-    def save_reduced_diffraction_data(self, hidra_project):
+    def save_reduced_diffraction_data(self, hidra_project, sub_runs):
         """ Export reduced diffraction data to project
         :param hidra_project: HidraProjectFile instance
         :return:
```

Comment:
can you and a docsstring for the new parameter?

Reply:
I added the docstring for the function. 

I did not add a test. I did offline testing to ensure that the function worked as expected. I can add a test if needed

Reply:
it'd be great if you could add an integration test 😁